modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -115,9 +115,12 @@ Your Amazon EC2 Spot Instances might be interrupted at any time. Use Amazon EC2
 ====
 If you select *Use Amazon EC2 Spot Instances* for a machine pool, you cannot disable the option after the machine pool is created.
 ====
-
+ifdef::openshift-rosa[]
+. Optional: Configure *Root disk size*.
```

Comment:
In "Add machine pool" section (day 2 operations), The Root disk size widget present below the compute node count widget. Hence this doc statement should be added as step 7.
See below reference
![image](https://github.com/openshift/openshift-docs/assets/93653094/890daf23-e190-423e-863b-6b3aa8e40062)